plugin.rb

```diff
@@ -44,4 +44,9 @@
   add_to_class(:topic_query, :list_docs_topics) do
     default_results(@options)
   end
+
+  on(:robots_info) do |robots_info|
+    any_user_agent = robots_info[:agents].find { |info| info[:name] == "*" }
```

Comment:
What happens if there are no "`*`" agent in the list of agents?

Reply:
That should be present by default. I updated the PR to cover this case anyway.

Reply:
Since any plugin can register to this event and alter `robots_info`, there's no "default" ;)